### PR TITLE
Add a guide on cloaks

### PR DESCRIPTION
**Pull request by jmbwell**
Addresses #52

### CODE REVIEW

**Comment by glguy** [comment]
Independent of how people will get cloaks in the future, the information here about them in general seems useful to share.

File: content/_guides/cloaks.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+The bot will give your account a generic user cloak and then — because its work
+is done — politely kick and ban you from the channel. Your new cloak takes
+effect immediately. 
+
+## Project or community cloaks
+
+If you are involved with a well-known project or community, you may request a
+cloak indicating your affiliation. For help with this, ask someone associated
+with your project or community who can coordinate with Libera.Chat to
+administer cloaks.
```
Comment (examknow) [inline content/_guides/cloaks.md:44]: we should probably specify that "group contacts" are the people who can request project cloaks

File: content/_guides/cloaks.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+You may see your own IP address when you `/whois` yourself. This is visible
+only to you and is not part of the hostmask visible to others.
+
+## Anonymity and privacy
+
+While a cloak helps limit the exposure of your IP address, other tools can
+more reliably provide a greater degree of privacy protection, including
+bouncers, cloud-based hosts, and Tor. To connect to Libera.Chat via Tor, see
+[Connecting to Libera.Chat](/guides/connect).
```
Comment (swantzter) [inline content/_guides/cloaks.md:66]: Might as well link to the section you're talking about?
```suggestion
[Connecting to Libera.Chat](/guides/connect#accessing-liberachat-via-tor).
```

File: content/_guides/cloaks.md (hunk 1 above)
Comment (jmbwell) [reply content/_guides/cloaks.md]: made an attempt at this

File: content/_guides/cloaks.md [hunk 3]
```diff
@@ -38,10 +38,9 @@ effect immediately.
 
 ## Project or community cloaks
 
-If you are involved with a well-known project or community, you may request a
-cloak indicating your affiliation. For help with this, ask someone associated
-with your project or community who can coordinate with Libera.Chat to
-administer cloaks.
+If you are involved with a registered [project](https://libera.chat/chanreg#project-registration) or [community](https://libera.chat/chanreg#community-registration), you may request a
+cloak indicating your affiliation. For help with this, ask a group
+contact for your project or community who can request cloaks.
```
Comment (examknow) [inline content/_guides/cloaks.md:43]: I *think* there should be a comma after community too

**Review by examknow — CHANGES_REQUESTED** [review]

File: content/_guides/cloaks.md (hunk 3 above)
Comment (jmbwell) [reply content/_guides/cloaks.md:45]: I debated. As a restrictive appositive, I don't think it calls for a comma, and I think I am going for restrictive appositive. Perhaps not all group contacts can request a cloak, in which case the procedure is to ask specifically a member of that subset of group contacts who can request cloaks. But it does work the other way, if we're just saying to ask anyone who can request a cloak, which would be any group contact, in which case, comma. Happy to go with the group consensus either way.

Comment (examknow) [reply content/_guides/cloaks.md:45]: I'm in no way an expert on the topic. I simply thought it seemed like one should go there :)

Comment (jmbwell) [reply content/_guides/cloaks.md:45]: Well, I never miss a chance to overthink it!

File: content/_guides/cloaks.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+is done — politely kick and ban you from the channel. Your new cloak takes
+effect immediately. 
+
+## Project or community cloaks
+
+If you are involved with a registered
+[project](https://libera.chat/chanreg#project-registration) or
+[community](https://libera.chat/chanreg#community-registration), you may
+request a cloak indicating your affiliation. For help with this, ask a group
+contact for your project or community who can request cloaks.
```
Comment (swantzter) [inline content/_guides/cloaks.md:45]: Maybe a note that if GC's have decided to be public you can find them with `/msg ChanServ INFO #projectchannel`

File: content/_guides/cloaks.md (hunk 3 above)
Comment (jmbwell) [reply content/_guides/cloaks.md]: fixed it by reworking the sentence anyway. thanks!

File: content/_guides/cloaks.md (hunk 4 above)
Comment (jmbwell) [reply content/_guides/cloaks.md]: made an attempt at this

**Comment by jmbwell** [comment]
Apologies — you can see some other work got pulled into this PR. I will fix it.

**Review by emersonveenstra — CHANGES_REQUESTED** [review]

File: content/_guides/cloaks.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+You can request a generic user cloak from the bot in `#libera-cloak`. The
+cloak will be in the form of `user/<NickServ username>`. Registration with
+`NickServ` is required.
+
+### Requesting a self-service user cloak
+
+Please note: This process may change in the future.
+
+1. [Register with NickServ](/guides/registration) if you have not already
+   done so. You will also need to be [identified](cloaks/).
```
Comment (emersonveenstra) [inline content/_guides/cloaks.md:40]: Should this be linking to https://libera.chat/guides/registration#logging-in ?

Comment (meldra) [reply content/_guides/cloaks.md:40]: lol yes oops

**Pull request merged**